packages/gatsby-admin/src/pages/index.tsx

```diff
@@ -1,5 +1,72 @@
 import React from "react"
-import { useQuery } from "urql"
+import { useQuery, useMutation } from "urql"
+
+const InstallInput = () => {
```

Comment:
The linter is complaining about missing return values, which is why it's failing tests. Type them as `React.FC` and you'll get that for free